src/YoutubeAtom.tsx

```diff
@@ -192,20 +210,31 @@ export const YoutubeAtom = ({
      */
     const showPlaceholder = !iframeSrc && (!hasOverlay || hasUserLaunchedPlay);
     /**
-     * Load the you tube iframe if:
-     *
-     * - We have a source string defined (i.e. We have consent)
-     *
-     * and
-     *
-     * - One of these 3 things are true
-     *      - We don't have an overlay - so we have to load the video straight away
-     *      - The user has clicked on the overlay, so load the video iframe!
-     *      - The user has moved their mouse over the overlay so lets pre load
-     *        the content
+     * Load the YouTube iframe if:
      */
-    const loadIframe =
-        iframeSrc && (!hasOverlay || hasUserHovered || hasUserLaunchedPlay);
+    let loadIframe: boolean;
+    const isMobile = /Mobi/.test(navigator.userAgent);
```

Comment:
I think this is more about testing touch v pointer than it is about mobile devices. Maybe the variable name should reflect that.

I know I was the one originally mentioning touch devices, but thinking about this again, I’m wondering if the best isn’t to just listen to a [`touchstart`][] event that would act as a `hover` state.

> It’s important to note that these types of optimisations based on the availability of touch have a fundamental flaw: they make assumptions about user behavior based on device capabilities. More explicitly, the example above assumes that because a device is capable of touch input, a user will in fact use touch as the only way to interact with it.
> 
> _from 2013: [Detecting touch: it's the 'why', not the 'how'](https://hacks.mozilla.org/2013/04/detecting-touch-its-the-why-not-the-how/)_

[`touchstart`]: https://developer.mozilla.org/en-US/docs/Web/API/Element/touchstart_event

Reply:
That's a really interesting, and probably very valid, point. I'm going to look into this some more (thanks for the links!) and chat it through with team 👍

Reply:
I discussed this with @OllysCoding and we both agreed your points are spot on @mxdvl and so I've refactored this PR to use `onTouchStart` in addition to `onHover`. The result of this is a my previously rather large PR is now basically a one-liner.

It's true that this means we sometimes delay the point at which a video is downloaded but as per your points and the linked articles I don't think we were accurately making the decisions about when to do this before and waiting until the user actually interacts is more accurate.